.github/ci-test-connections.yaml

```diff
@@ -73,3 +73,17 @@ connections:
     extra:
       aws_access_key_id: $AWS_ACCESS_KEY_ID
       aws_secret_access_key: $AWS_SECRET_ACCESS_KEY
+  - conn_id: minio_conn_1
+    conn_type: aws
+    description: null
+    extra:
+      aws_access_key_id: "ROOTNAME"
+      aws_secret_access_key: "CHANGEME123"
+      host: "http://127.0.0.1:9000"
```

Comment:
what about using ```endpoint_url``` instead of host since ```host``` in extra has deprecated in Airflow aws conn?

Reply:
@pankajastro We are testing it with both actually. `minio_conn_2` has `endpoint_url`

Reply:
ok, just curious why testing for both i.e host and endpoint_url

Reply:
@pankajastro  We needed that earlier since we were passing the values like endpoint_url while creating a client, and the filed name has changed in apache-airflow-providers-amazon from `host` to `endpoint_url` in the 5.0.0 version and we wanted to handle both cases. Now we don't need that since we are using provide a method to do that.